.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,53 +1,42 @@
-## Description
-
-<!--- 
-Describe the purpose of this PR along with any background information and the impacts of the proposed change. 
-For the benefit of the community, please do not assume prior context.
-
-Provide details that support your chosen implementation, including: 
-- breaking changes
-- alternatives considered
-- changes to the API
-- demos (screenshots, videos) if you find that useful
-- etc.
+<!--
+❗ For general support or usage questions, use the Auth0 Community forums or raise a support ticket.
+
+By submitting a pull request to this repository, you agree to the terms within the Auth0 Code of Conduct: https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md.
 -->
 
+### 🔧 Changes
+
+<!--
+Describe both what is changing and why this is important. Include:
+
+- Types and methods added, deleted, deprecated, or changed
+- A summary of usage if this is a new feature or a change to a public API
+-->
 
-## References
+### 📚 References
 
-<!--- 
-Include any links supporting this change such as a:
+<!--
+Add relevant links supporting this change, such as:
 
-- GitHub Issue/PR number addressed or fixed
+- GitHub issue/PR number addressed or fixed
 - Auth0 Community post
-- StackOverflow post
-- Support forum thread
-- Related pull requests/issues from other repos
+- StackOverflow answer
+- Related pull requests/issues from other repositories
 
 If there are no references, simply delete this section.
 -->
 
+### 🔬 Testing
 
-## Testing
-
-<!--- 
-Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. 
-If this library has unit and/or integration testing, tests should be added for new functionality and 
-existing tests should complete without errors.
+<!--
+Describe how this can be tested by reviewers. Be specific about anything not tested and why. Include any manual steps for testing end-to-end, or for testing functionality not covered by unit tests.
 -->
 
-- [ ] This change adds test coverage for new/changed/fixed functionality
-
+### 📝 Checklist
```

Comment:
I'd suggest moving this to the top, right above "Changes", so it's the first thing people see, and they don't waste time filling in the previous fields only to find out they still can't raise the PR.